homeassistant/components/powerwall/sensor.py

```diff
@@ -133,7 +133,7 @@ def _get_meter_average_voltage(meter: MeterResponse) -> float:
         state_class=SensorStateClass.MEASUREMENT,
         device_class=SensorDeviceClass.VOLTAGE,
         native_unit_of_measurement=UnitOfElectricPotential.VOLT,
-        value_fn=lambda battery_data: round(battery_data.v_out, 1),
+        value_fn=lambda battery_data: round(battery_data.v_out or 0.0, 1),
```

Comment:
In these cases, why don't you remove the round and set a `suggested_precision`?

Reply:
In a separate PR